lib/templates/deploy.bash.ruby.template

```diff
@@ -33,19 +33,32 @@ echo "$DEPLOYMENT_TARGET"
 if [ -e "$DEPLOYMENT_TARGET/Gemfile" ]; then
   echo "Found gemfile"
   pushd "$DEPLOYMENT_TARGET"
+  
   eval "$(rbenv init -)"
   exitWithMessageOnError "init failed"
+  
   echo "Setting ruby version"
   rbenv global $WEBSITES_LATEST_NODE_VERSION
   exitWithMessageOnError "Failed to switch ruby versions"
-  bundle config --local path "vendor/bundle"
+  
   echo "Running bundle clean"
   bundle clean --force
+  
   echo "Running bundle install"
-  bundle install --path "vendor/bundle" $OPTIONS
-  echo "Running bundle package"
-  bundle package --all
+  mkdir -p /tmp/bundle
```

Comment:
Still want to look at this in detail and understand it, but paths should always be parameterized, even if we have to modify kuduscript itself